### PR TITLE
Revert to singular CLI args

### PR DESCRIPTION
**Pull request by qwerty287**
revert parts of https://github.com/woodpecker-ci/woodpecker/pull/2213

### CODE REVIEW

**Comment by xoxys** [comment]
https://github.com/woodpecker-ci/woodpecker/blob/main/docs/docs/91-migrations.md?plain=1#L9 needs to be reverted as well

File: cli/secret/secret_set.go [hunk 1]
```diff
@@ -46,11 +46,11 @@ var secretUpdateCmd = &cli.Command{
...
 		},
 		&cli.StringSliceFlag{
-			Name:  "events",
+			Name:  "event",
 			Usage: "secret limited to these events",
 		},
 		&cli.StringSliceFlag{
-			Name:  "images",
+			Name:  "image",
 			Usage: "secret limited to these images",
```
Comment (xoxys) [inline cli/secret/secret_set.go:54]: Not related to this PR but the `Usage` seems to be a copy/paste one and is wrong.

Comment (xoxys) [reply cli/secret/secret_set.go:54]: Ah no... My bad

**Review by 6543 — APPROVED** [review]

**Comment by anbraten** [comment]
Was there an issue with the renamings?

**Comment by 6543** [comment]
it was discussted at maintainers channel via chat

tldr: this means `--evens cron` `--events tags` instead of `--event cron` `--event tags`

we can always have alias if that is relay a problem
